### ISSUE TITLE
Language name

### ISSUE THREAD

**Issue by monfera** [julianhyde/morel#80]
A (the?) key language name quality is, unique searchability. 

Languages named helpfully for language learners and users (hello Stack Overflow):
- Clojure, ClojureScript
- JavaScript
- Kotlin
- SQL
- GraphQL
- Datomic
- LINQ

Ambiguous names with previous and ongoing confusion, or too short to be unique:
- Go
- Rust
- C
- Python

Maybe not too late to consider a name more unique than Morel. It'd add possibilities like the use of Google Trends, and search of future references in academic papers

**Comment by julianhyde** [comment]
I agree. Naming is important. It's also hard.

Fun fact. About 20 years ago I was working on Mondrian, an implementation of the MDX (multi-dimensional expressions) language. It was easy to find examples until Acura launched an SUV of the same name.

Also in my younger years, I played guitar in a number of bands. Choosing a name was always the hardest part. Some of the bands broke up before we managed to agree on a name.

Morel has already been renamed once. It used to be smlj, which was very searchable but not very pronounceable, and didn't convey the emphasis on relational algebra.

Maybe it will be renamed again. Or better still, someone will fork it and it will evolve into something better. (BCPL yielded B yielded C; Oak was evolved into Java.)
